cli/csstools-cli/CHANGELOG.md

```diff
@@ -3,12 +3,16 @@
 ### Unreleased (minor)
 
 - Emit warnings from plugins
-- Added `@csstools/postcss-is-pseudo-class`
+- Added `@csstools/postcss-font-format-keywords`
```

Comment:
This also needs a change in `cli/csstools-cli/src/cli.ts`

1. add the plugin in the `switch` statement
2. add the key from the `switch` statement in the general help output

The general help also exists in the README, but this doesn't always have to be up to date as it's an example

Reply:
Thanks for reviewing that! I've added those!